### PR TITLE
fix some Javadoc warnings

### PR DESCRIPTION
**Pull request by elharo**
@briandealwis

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/StringVersionComparator.java [hunk 1]
```diff
@@ -19,7 +19,7 @@
 import java.util.Comparator;
 
 /**
- * A {@link Comparator<String>} for the Faceted Project framework that compares versions that are
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/StringVersionComparator.java]: Why not a `@link` here?

File: plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/CloudSdkDebugTargetPresentation.java [hunk 2]
```diff
@@ -32,7 +32,7 @@
 
 /**
  * A Platform/Debug Presentation to provide an icon for our
- * {@link LocalAppEngineServerLaunchConfigurationDelegate.CloudSdkDebugTarget} instances.
+ * {@code LocalAppEngineServerLaunchConfigurationDelegate.CloudSdkDebugTarget} instances.
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/CloudSdkDebugTargetPresentation.java]: Why not a `@link` here?

File: plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/MajorVersionPipelineRunnerConfigurationTest.java [hunk 3]
```diff
@@ -31,7 +31,7 @@
 
 /**
  * Test that each {@link MajorVersion} has associated configuration settings in the
- * {@link PipelineRunner#RUNNERS_IN_VERSION} mapping table and {@link PipelineLaunchConfiguration}.
+ * {@code PipelineRunner#RUNNERS_IN_VERSION} mapping table and {@link PipelineLaunchConfiguration}.
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/MajorVersionPipelineRunnerConfigurationTest.java]: Ditto?

File: plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/StringVersionComparator.java (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/StringVersionComparator.java]: I think Javadoc isn't smart enough to link to generics from an `{@ link}`. In any case, Eclipse flags this as a warning.

File: plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/CloudSdkDebugTargetPresentation.java (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/ui/CloudSdkDebugTargetPresentation.java]: not imported

File: plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/MajorVersionPipelineRunnerConfigurationTest.java (hunk 3 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.dataflow.core.test/src/com/google/cloud/tools/eclipse/dataflow/core/launcher/MajorVersionPipelineRunnerConfigurationTest.java]: private and thus can't be linked to

**Pull request merged**